shakenfist/node.py

```diff
@@ -130,3 +132,27 @@ def __iter__(self):
     baseobject.state_filter, [dbo.STATE_CREATED])
 inactive_states_filter = partial(
     baseobject.state_filter, [dbo.STATE_DELETED, dbo.STATE_ERROR, 'missing'])
+
+
+def _sort_by_key(d):
+    for k in sorted(d):
+        for v in d[k]:
+            yield v
+
+
+def nodes_by_free_disk_descending(minimum=0, maximum=-1):
+    by_disk = {}
```

Comment:
another place to use `defaultdict(list)`

Reply:
Good catch.